packages/ui-react/src/components/TruncatedText/TruncatedText.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { axe } from 'vitest-axe';
+import { render } from '@testing-library/react';
+
+import TruncatedText from './TruncatedText';
+
+describe('<TruncatedText>', () => {
+  test('renders and matches snapshot', () => {
+    const { container } = render(<TruncatedText text="Test..." maximumLines={8} />);
+
+    expect(container).toMatchSnapshot();
+  });
+
+  test('WCAG 2.2 (AA) compliant', async () => {
+    const { container } = render(<TruncatedText text="Test..." maximumLines={8} />);
+
+    expect(await axe(container, { runOnly: ['wcag21a', 'wcag21aa', 'wcag22aa'] })).toHaveNoViolations();
```

Comment:
Even though the intention is great, this test doesn't add a lot of value because it's only a `div` with text in it. This cannot go wrong. Do you agree?

Reply:
Hmm yes it might be a bit redundant, I agree, I'll remove this one